Doc 1496/changelog (#532)
swan-io/docs: changelog/2026/2026-01-15.mdx

---
title: 15 January 2026
type: release
---

## New features

📖 Subscribe to email notifications directly from the [Changelog](/changelog) to stay informed about API updates and breaking changes.

## Improvements

📄 The [`generateTransactionStatement` mutation](https://api-reference.swan.io/mutations/generate-transaction-statement/) now works for incoming and outgoing `Booked` [International Credit Transfers](/topics/payments/credit-transfers/international/). Use it to prove a transaction was executed by Swan.

## API updates

### Upcoming breaking changes

> 🌍 Starting January 19, a new `accountId` field is available as an optional parameter in [`internationalBeneficiaryDynamicForms`](https://api-reference.swan.io/queries/international-beneficiary-dynamic-forms/). It becomes mandatory on **March 31, 2026**. Update your integration during this period to ensure continued support for international credit transfer queries.

Reviewer: ```suggestion
🌍 Starting January 19, a new `accountId` field is available as an optional parameter in [`internationalBeneficiaryDynamicForms`](https://api-reference.swan.io/queries/international-beneficiary-dynamic-forms/). This field becomes mandatory on March 31, 2026. Update your integration before this date to ensure continued support for international credit transfer queries.
```

Reviewer: Makes it more clear the impact after March 31.

### Reminders

> 📆 On **April 1, 2026**, the following fields will be removed from the [`accountStatement`](https://api-reference.swan.io/queries/account-statement/) and [`Account.statements`](https://api-reference.swan.io/objects/account/) queries and from the [`generateAccountStatement`](https://api-reference.swan.io/mutations/generate-account-statement/) mutation response:

Reviewer: ```suggestion
📆 On **April 1**, the following fields will be removed from the [`accountStatement`](https://api-reference.swan.io/queries/account-statement/) and [`Account.statements`](https://api-reference.swan.io/objects/account/) queries and from the [`generateAccountStatement`](https://api-reference.swan.io/mutations/generate-account-statement/) mutation response:
```
